packages/mobile/src/account/saga.ts

```diff
@@ -97,6 +103,21 @@ function* migrateAccountToProperBip39() {
   }
 }
 
+function* clearStoredAccountSaga({ account }: ClearStoredAccountAction) {
+  try {
+    yield call(removeAccountLocally, account)
+    yield call(clearStoredMnemonic)
+    yield call(ValoraAnalytics.reset)
+    yield call(firebaseSignOut, firebase.app())
+    // TODO: Remove the account from geth.
+
+    yield call(persistor.flush)
+    yield call(restartApp)
+  } catch (error) {
+    Logger.info(TAG + '@clearStoredAccount', 'Error while removing account', error)
```

Comment:
We should surface this error to the user somehow. I'd recommend using `yield put(showError(...` like we do in many sagas.